srcpkgs/taskd/files/taskd/run

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 [ -r conf ] && . ./conf
-: ${TASKDDATA:=/var/lib/taskd}
-: ${TASKDUSER:=taskd}
+TASKDDATA="/var/lib/taskd"
+TASKDUSER=taskd
 exec 2>&1
-exec chpst -u ${TASKDUSER} taskd --data ${TASKDDATA}
+exec taskd server --daemon --data ${TASKDDATA} user ${TASKDUSER}
```

Comment:
I think `--daemon` wrong, we explicitly don't want that the process daemonizes for runit.

Reply:
K, Will fix